redpanda/values.schema.json

```diff
@@ -33,8 +33,15 @@
       }
     },
     "license_key": {
-      "type": "string",
-      "pattern": "^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$"
+      "type": "object",
```

Comment:
Let's have both, the `license_key` and a `license_secret_ref`. Mark the `license_key' deprecated.